plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergAvroConnectorTest.java

```diff
@@ -1,51 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.trino.plugin.iceberg;
-
-import org.testng.SkipException;
-
-import static io.trino.plugin.iceberg.IcebergFileFormat.AVRO;
-
-public class TestIcebergAvroConnectorTest
-        extends BaseIcebergConnectorTest
```

Comment:
`BaseIcebergConnectorTest` contains many tests which are specific to file format. You can find "AVRO" keyword in the class. I believe this change decreases test coverages.

Reply:
fair point. The question is: does it decrease it substantially?

Reply:
@ebyhr @alexjo2144 @findinpath thoughts?

Reply:
I guess we will lose the test coverage of Avro statistics because the smoke test doesn't cover such an area. If we cover them in the smoke test, we can safety remove `TestIcebergAvroConnectorTest` class in my opinion.